README.md

```diff
@@ -1 +1,27 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## About Me
+
+Hello! My name is **Nikhil Mandalaparthy** (my first name is pronounced *nick-hill*). I'm currently a MPP student at the Harris School of Public Policy, and I just graduated in June from the College, with a BA in Public Policy Studies. This is my [personal website](https://nikhilmandala.com/).
+
+## My Summer
+
+This summer, I was lucky enough to spend time in different cities, countries, and continents! Here's what I was up to:
+
+- After graduation, I went back home to **Seattle, WA** for a week.
+- Then, I went across the country to **Washington, DC** for an internship.
+- Following my internship, I went back to Seattle and then helped my family move to **San Jose, CA**.
+- I then went to India for a journalism project. I spent most of my time in the national capital of **New Delhi**, and I also visited a smaller city in the northwest called **Ajmer**.
+- Now I'm back in **Chicago** for my last year at UChicago.
```

Comment:
Great list. Potentially could've used sub-bullets since it's clear you're a capable user of RMarkdown